README.md

```diff
@@ -361,8 +361,6 @@ All of these different pieces are available and linked together properly within
 
 We use Docker throughout this build process to both install the Forseti Security libraries and process the code. This reduces the amount of dependencies necessary on the host machine to *only* Docker. There's no need for Python, Sphinx, or anything else to be modified on the host machine.
```

Comment:
Please remove `Sphinx` in this sentence.

Reply:
Done